spec/models/post_spec.rb

```diff
@@ -0,0 +1,18 @@
+RSpec.describe Post, type: :model do
+  describe '#recent_comments' do
+    let(:user) { User.create(name: 'Joy', photo: 'https://www.google.com/url', bio: 'she is a software developer', posts_counter: 0) }
+    let(:post) do
+      Post.create!(author_id: user.id, title: 'Post 1', text: 'This is post 1', comments_counter: 0, likes_counter: 0)
+    end
+
+    it 'returns 5 most recent comments for a post' do
+      6.times { |i| Comment.create(post:, user:, text: "Comment #{i + 1}") }
+
+      recent_comments = post.reload.comments.order(created_at: :desc).limit(5)
+
+      expect(recent_comments.count).to eq(5)
+      expect(recent_comments.first.text).to eq('Comment 6')
+      expect(recent_comments.last.text).to eq('Comment 2')
+    end
```

Comment:
- [ ] Great progress so far! 👏 Your recent work aligns well with the project requirements. However, I noticed that the unit tests for the added validations in the model are currently missing. As per the project guidelines, it's crucial to have comprehensive test coverage to ensure the reliability of your code.

     - Here's a friendly nudge to consider adding tests for the specific validations highlighted in the attached screenshot. 🧐 Each test can be encapsulated within its own `it` block, which not only adheres to best practices but also makes the tests more readable and easy to understand. This small adjustment enhances the overall maintainability of your test suite.

<img width="661" alt="Screenshot 2023-12-09 at 06 39 06" src="https://github.com/Ayokunnumi1/blog_app/assets/107192076/b4863e58-1623-4137-ab9d-f84a61bf7be5">

 Happy coding! 🚀👨‍💻😊